app/views/groups/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="header-div">
+   <h1 class="header"> Expenditures </h1>
+</div>
+<div id="groups" class="container">
+<% if user_signed_in? %>
+ <div class="grouping">
+ <%= button_to 'Sign Out', destroy_user_session_path, method: :delete %>
+
```

Comment:
- [ ] Kindly add a back button to your page so users can easily navigate back to the home page, this will create a better user experience 👍🏿